### DISCUSSION TITLE
(Projects Beta) No way to return to organization/repo

### DISCUSSION THREAD

**Discussion by lisa-wolfgang** [community/community#9551] [Projects and Issues]
On the beta project boards, there's no UI to return to the organization or repo the project board was opened from. Furthermore, if the user has been making a lot of changes to the project, not even the back button on their browser will help because GitHub generates a new link for every change to the project view.

I understand that the org/repo headers take up a lot of vertical space, but at the very least, there should be a back button to the left of the project title.

**Comment by MisinformedDNA** [comment]
Tip: If you add the "Repository" column, the repository will be clickable.

But, I agree, there really needs to be a header to get back to the org (at the very least).

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
